src/Countries/Switzerland.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+
+class Switzerland extends Country
+{
+    public function countryCode(): string
+    {
+        return 'ch';
+    }
+
+    protected function allHolidays(int $year): array
+    {
+        return array_merge([
+            'Neujahr' => '01-01',
+            'Berchtoldstag' => '01-02',
```

Comment:
This is not a public holiday in every canton. In the resource (wikipedia) you can see that this day is not official everyware. 
F.e. Kt. BE does not celebrate this day.

Reply:
<img width="1292" alt="image" src="https://github.com/spatie/holidays/assets/26961559/334ad1db-c0b5-4f67-aa1b-6bb2b4cb0c92">
@herpaderpaldent this PR covers only the non-regional holidays until now. You can read the description of the PR which days are included and why.

Reply:
Hey @Martin-Welte i am sorry i can not see a description to this PR that would have explained this. However the simplification is understandable until the regional support is settled. 

May i suggest changing the source towards the original source and not wikipedia?